controller/DeliveryTool.php

```diff
@@ -61,6 +62,14 @@ class DeliveryTool extends ToolModule
      * @var string
      */
     const PARAM_SKIP_THANKYOU = 'custom_skip_thankyou';
+
+    /**
+     * Setting this parameter to 'true' will prevent the 'You have already taken this test'
+     * screen to be shown skip directly to the return url
+     * @var string
+     */
+    const FORCE_REDIRECT_TO_RETURN_URL = 'custom_force_redirect_to_return_url';
```

Comment:
It seems that the convention for constants are `PARAM_*`. And the name of parameter seems misleading for me, it does not force much, f.e. if test taker never started the delivery then there is no redirect. I'd suggest `PARAM_SKIP_OVERVIEW`